src/intro/intro_readme.md

```diff
@@ -1,4 +1,4 @@
-![header](./img/intro_header.png)
+![header](img/intro_header.png)
```

Comment:
This wasn't working when you opened the manual from the short url: www2.manual.grid.tf

Thus in a commit from today, I changed it to this:
![header](../intro/img/intro_header.png)

Now it works all the time. I added a commit to this PR to make it work without conflicts.

Reply:
will make sure next time its coded like that, thanks @Mik-TF